rkt/rkt.go

```diff
@@ -105,6 +111,11 @@ func init() {
 	cmdRkt.PersistentFlags().BoolVar(&globalFlags.TrustKeysFromHttps, "trust-keys-from-https",
 		true, "automatically trust gpg keys fetched from https")
 
+	// Run this before the execution of each subcommand to set up output
+	cmdRkt.PersistentPreRun = func(cmd *cobra.Command, args []string) {
+		errOut = common.NewErrorSanitizer(os.Stderr, cmd.Name(), globalFlags.Debug)
```

Comment:
right. We need to separate the idea of making the errors structured and outputting the errors. For stage1 binaries, the errors have now been structured but I haven't dealt with output part yet. A large majority, if not all, of the output for stage1 goes to log.

Currently the wrapped errors will only show the outer most message. Will probable need to change that.